hudi-cli/src/main/java/org/apache/hudi/cli/commands/ArchivedCommitsCommand.java

```diff
@@ -53,11 +52,6 @@
 @Component
 public class ArchivedCommitsCommand implements CommandMarker {
 
-  @CliAvailabilityIndicator({"show archived commits"})
```

Comment:
`CliAvailabilityIndicator` can not give us any useful hint, it can just return `false` when we try to throw exception. In `CliAvailabilityIndicator ` source code, it catch any exception and return `false`.

Reply:
ok